### DISCUSSION TITLE
SEVI作业中R_0的值如何确定？

### DISCUSSION THREAD

**Discussion by May0703** [physics-data/faq#81] [Q&A] [answered]
R_0和\delta_R的值都应该如何确定呢？readme文档中并没有写明。

**Comment by Harry-Chen** [comment]
@littlewu2508

**Comment by littlewu2508** [comment]
R_0 就是最终电子球面的半径。鉴于我们的屏幕是 [-1,1]*[-1,1] 的，我们选取 R_0 = 0.8 应该会比较好看。

**Comment by littlewu2508** [comment, marked as answer]
我在 https://github.com/physics-data/tpl_SEVI/commit/8f08e94e7faad1e06973615183bcbec7628930f0 更新了 R0 与 DeltaR 两个参数。已经生成自己repo的，可以添加这个原始仓库的远端（例如名为 "upstream"），git fetch，然后有多种方案：
1. 将 upstream/master git rebase 到自己仓库的 master 分支上；
2. git cherry-pick 8f08e94e7faad1e06973615183bcbec7628930f0, 直接将这个 commit 作用到自己的 master 分支上。
操作完成后本地的 README.md 会更新这俩参数。
